migration/identity-migration/src/main/java/io/camunda/migration/identity/config/oidc/StaticEntities.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.identity.config.oidc;
+
+import io.camunda.migration.identity.config.oidc.OidcProperties.Audience;
+import io.camunda.service.AuthorizationServices.CreateAuthorizationRequest;
+import io.camunda.zeebe.protocol.record.value.AuthorizationOwnerType;
+import io.camunda.zeebe.protocol.record.value.AuthorizationResourceType;
+import io.camunda.zeebe.protocol.record.value.PermissionType;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class StaticEntities {
+
+  private static final String IDENTITY_RESOURCE_ID = "identity";
+  private static final String OPERATE_RESOURCE_ID = "operate";
+  private static final String TASKLIST_RESOURCE_ID = "tasklist";
+
+  public static List<CreateAuthorizationRequest> getAuthorizationsByAudience(
+      final Audience properties,
+      final String audience,
```

Comment:
🔧 the second param is actually the original permission, a composite of audience and permission value, right?
```suggestion
      final Audience audiences,
      final String permission,
```